Разметка всех страниц

Statsou opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2187575/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2187575-sedona-35/2/

Statsou committed:
Добавляет шапку

Statsou committed:
Централизирует сайт

Сайт больше не уезжает влево при изменении масштаба при этом, сохраняет нужные внешние отступы
Изменены иконки на svg.

Statsou committed:
Delete number.svg

Statsou committed:
Update style.css

Statsou committed:
Добавляет элементы в мэйн

Statsou committed:
Update style.css

Statsou committed:
Добавляет грид в мэйн

Statsou committed:
Выравнивает грид и отступы

Statsou committed:
Выравнивает отступы

Выравнивает отступы на всей первой странице

Statsou committed:
Добавляет разметку index и catalog

Добавляет разметку 2-ух страниц и стилизация страницы index

Statsou committed:
Update style_header.css

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Каталог</title>
+    <link rel="stylesheet" href="/style/style_header.css">
```
у нас будет один стилевой файл на обе страницы

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+    <div class="content">
+      <header>
+        <nav class="navigation">
+          <div class="container-left">
```
в названии класса не нужно привязываться к расположению или внешнему виду блока

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+            <a href="index.html">
+              <img src="images/logo.svg" alt="logo" class="logo">
+            </a>
+              <ul class="navigation-list">
```
лишний отступ слева

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+          </div>
+          <ul class="container-right">
+            <li>
+              <img src="images/icons/search.svg" alt="Поиск" class="search">
```
тут будет ссылка

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+              <img src="images/icons/search.svg" alt="Поиск" class="search">
+            </li>
+            <li>
+              <img src="images/icons/heart.svg" alt="Избранное" class="heart">
```
тут будет ссылка

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+              <span class="new-favorites">12</span>
+            </li>
+            <li>
+              <button class="button want">Хочу сюда!</button>
```
это ссылка

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                </li>
+                <li class="navigation-item current">
+                  <a href="#">Гостиницы</a>
+                  <hr class="hr-current catalog">
```
это лишнее

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+          <ul class="breadcrumbs">
+            <li class="breadcrumbs-item">
+              <a href="index.html" class="breadcrumbs-link">
+                <img src="images/icons/in_home.svg" alt="На главную" class="breadcrumbs-in-home">
```
класс должен быть `breadcrumbs-image`

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                  </li>
+                </ul>
+              </fieldset>
+              <button class="butto blue" type="submit">Применить</button>
```
catalog-filter-submit-button

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                </ul>
+              </fieldset>
+              <button class="butto blue" type="submit">Применить</button>
+              <button class="reset" type="reset">Сбросить</button>
```
catalog-filter-reset-button

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+            </a>
+            <a href="#" class="button-light">
+              <span class="visuality-hidden">Показать список</span>
+            </a>
```
объедини эти однородные элементы в список

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+        <section class="catalog">
+          <ul class="product-cards">
+            <li class="product-card">
+              <a class="product-card-link" href="#">
```
заголовок и картинка должны быть обернуты отдельными ссылками, одна для заголовка, другая для картинки

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                <img src="/images/product-card/hotel-1.jpg" class="product-card-image" alt="Фотография отеля Amara Resort & Spa">
+                <h3>Amara Resort & Spa</h3>
+              </a>
+                <section>
```
тут не нужен тег section

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                <h3>Amara Resort & Spa</h3>
+              </a>
+                <section>
+                  <span class="type">Гостиница</span>
```
нужно задавать полные и понятные названия классов
тут будет класс product-card-type

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                  <span class="price">От 4000 ₽</span>
+                </section>
+                <a href="#" class="product-card-button butto">Подробнее</a>
+                <button class="favorites-off">Избранное</button>
```
product-card-button , в названии класса не нужно привязываться к тексту внутри элемента, ведь текст всегда может измениться

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                </section>
+                <a href="#" class="product-card-button butto">Подробнее</a>
+                <button class="favorites-off">Избранное</button>
+              <section class="raiting">
```
ну нежна тут секция

Statsou replied:
Если вы имели в виду кнопки, то они же вне секции

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                <img src="/images/icons/raiting-star.svg" class="star" alt="Оценка качества">
+                <img src="/images/icons/raiting-star.svg" class="star" alt="Оценка качества">
+                <img src="/images/icons/raiting-star.svg" class="star" alt="Оценка качества">
+              </span>
```
однородные элементы лучше делать списком

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                <img src="/images/icons/raiting-star.svg" class="star" alt="Оценка качества">
+                <img src="/images/icons/raiting-star.svg" class="star" alt="Оценка качества">
+              </span>
+              <span class="number-raiting">Рейтинг 8.5</span>
```
product-card-rating

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+          </ul>
+        </section>
+        <section class="subscribe">
+          <h2 class="visuality-hidden">Подписка на рассылку</h2>
```
не нужен тут скрытый заголовок, ведь ниже идёт текст точно с таким же содержимым

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+        </section>
+        <section class="subscribe">
+          <h2 class="visuality-hidden">Подписка на рассылку</h2>
+          <p class="subscribe-top">Подпишитесь на рассылку</p>
```
subscribe-title

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+        <section class="subscribe">
+          <h2 class="visuality-hidden">Подписка на рассылку</h2>
+          <p class="subscribe-top">Подпишитесь на рассылку</p>
+          <p class="subscribe-bot">Только полезная информация и никакого спама,<br> честное бойскаутское!</p>
```
subscribe-description

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+          <p class="subscribe-bot">Только полезная информация и никакого спама,<br> честное бойскаутское!</p>
+          <form action="https://echo.htmlacademy.ru/" method="post">
+              <input type="email" name="email" placeholder="Ваш e-mail" class="email">
+              <input type="submit" value="Подписаться" name="e-mail" class="subscribe-button">
```
разметь тегом button

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+        </section>
+      </main>
+      <footer class="page-footer">
+        <ul class="contact-messenger">
```
`social`

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+      </main>
+      <footer class="page-footer">
+        <ul class="contact-messenger">
+          <li>
```
social-item

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+      <footer class="page-footer">
+        <ul class="contact-messenger">
+          <li>
+            <a href="https://vk.com/htmlacademy"><img src="pictures/messenger/Vk.svg" alt="VK"></a>
```
social-link

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+          </li>
+        </ul>
+        <a href="tel:+7(812)812-12-12" class="contact-phone">+7 (812) 812-12-12</a>
+        <a href="https://htmlacademy.ru">
```
нужно задать класс

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+      <header>
+        <nav class="navigation">
+          <div class="container-left">
+            <a>
```
нужно задать класс

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+      </header>
+      <main>
+        <div class="welcome">
+          <h1 class="visuality-hidden">Добро пожаловать в Седону</h1>
```
`visually-hidden`

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+        <div class="welcome">
+          <h1 class="visuality-hidden">Добро пожаловать в Седону</h1>
+          <img src="images/welcome/welcome_sedona.svg" alt="Добро пожаловать в Седону" class="img-welcome">
+          <img src="images/welcome/vector-foot.svg" alt="Vector" class="img-welcome-vector" >
```
эта картинка, как и картинка с горами, будет фоном

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          <img src="images/welcome/welcome_sedona.svg" alt="Добро пожаловать в Седону" class="img-welcome">
+          <img src="images/welcome/vector-foot.svg" alt="Vector" class="img-welcome-vector" >
+        </div>
+        <section>
```
нужно задать класс

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          <img src="images/welcome/vector-foot.svg" alt="Vector" class="img-welcome-vector" >
+        </div>
+        <section>
+          <h2 class="bold-text top">Седона — небольшой городок в Аризоне,<br>
```
а потом от класса секции задать тут класс, [some-name]-title

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+        <section>
+          <h2 class="bold-text top">Седона — небольшой городок в Аризоне,<br>
+          заслуживающий большего!</h2>
+          <p class="normal-text top">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
```
[some-name]-description

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          <p class="normal-text top">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+        </section>
+        <ul class="advantages-list">
+          <li class="card blue">
```
advantages-item

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+        </section>
+        <ul class="advantages-list">
+          <li class="card blue">
+            <h3 class="content-blue">Настоящий Городок</h3>
```
advantages-title

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+        <ul class="advantages-list">
+          <li class="card blue">
+            <h3 class="content-blue">Настоящий Городок</h3>
+            <hr>
```
лишний тег

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          <li class="card blue">
+            <h3 class="content-blue">Настоящий Городок</h3>
+            <hr>
+            <p class="bottom-text">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
```
advantages-text или advatages-description

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+            <hr>
+            <p class="bottom-text">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+          </li>
+          <li class="card-image">
```
элементами списка могут быть только однородные элементы, картинка ведь не однородна с заголовком и параграфом. эта картинка будет внутри первого тега li

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          <li class="card-image">
+            <img src="images/card/card-1.jpg" alt="card" width="800" height="384">
+          </li>
+          <li class="card white-12">
+            <h3>Небольшая площадь</h3>
+            <hr class="white">
+            <p class="bottom-text">Все достопримечательности находятся очень близко</p>
+          </li>
+          <li class="card white-20">
+            <h3>Красивая дорога</h3>
+            <hr class="white">
+            <p class="bottom-text">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="card white-12">
+            <h3>Мало туристов</h3>
+            <hr class="white">
+            <p class="bottom-text">Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
```
вот эти три лишки будут вложенным списком, внутри первого тега li главного списка, этот тег будет закрываться после этого вложенного списка

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+            <hr>
+            <p class="bottom-text">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </li>
+          <li class="card white-12">
+            <img src="images/card/icons/house.svg" alt="icon" class="icon-grid">
+            <h3 class="bottom">Жильё</h3>
+            <p class="bottom-text">Рекомендуем пожить <br> в настоящем мотеле, <br>
+              всё как в кино!</p>
+          </li>
+          <li class="card">
+            <img src="images/card/icons/food.svg" alt="icon" class="icon-grid">
+            <h3 class="bottom">Еда</h3>
+            <p class="bottom-text">Всегда заказывайте <br> топовый фирменный бургер, вы не разочаруетесь!</p>
+          </li>
+          <li class="card white-12">
+            <img src="images/card/icons/box.svg" alt="icon" class="icon-grid">
+            <h3 class="bottom">Сувениры</h3>
+            <p class="bottom-text">Не только китайского, <br> но и настоящего местного производства!</p>
+          </li>
```
а эти три лишки по аналогии будут вложенным списком во второй тег li главного списка

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          </li>
+        </ul>
+        <div class="search-hotel">
+          <p class="bold-text hotel">Заинтересовались?</p>
```
search-hotel-text

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+        </ul>
+        <div class="search-hotel">
+          <p class="bold-text hotel">Заинтересовались?</p>
+          <p class="normal-text hotel">Укажите предполагаемые даты поездки,<br>
```
search-hotel-description

GasparColor commented on index.html:
```diff
@@ -3,10 +3,142 @@
+          <p class="bold-text hotel">Заинтересовались?</p>
+          <p class="normal-text hotel">Укажите предполагаемые даты поездки,<br>
+            и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          <button class="button hotel">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
```
search-hotel-button , и тут у кнопки тип неправильный

GasparColor commented on style/style_index.css:
```diff
@@ -0,0 +1,222 @@
+}
+
+.card.blue {
+color: #FFFFFF;
```
отступы везде должны быть единообразными

Statsou committed:
Исправляет списки и классы

1. Исправляет некоторые кнопки.
2. Исправляет основной грид в мэйне
3. Исправляет классы

Statsou committed:
Убирает лишний отступ слева

Statsou committed:
Добавляет классы

Statsou committed:
Добавляет класс к выбору типа каталога

Statsou committed:
Исправляет структуру документа

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+    <div class="content">
+      <header>
+        <nav class="navigation">
+          <div class="logo-menu">
```
лучше подойдет класс main-menu

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+        <nav class="navigation">
+          <div class="logo-menu">
+            <a href="index.html">
+            <img src="images/logo.svg" width="140" height="70"  alt="Логотип Седона." class="logo">
```
слева нужен отступ

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+      <header>
+        <nav class="navigation">
+          <div class="logo-menu">
+            <a href="index.html">
```
нужно задать класс

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+            </a>
+            <ul class="navigation-list">
+              <li class="navigation-item">
+                <a href="index.html" class="navigation-page">Главная</a>
```
navigation-link

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+            </ul>
+          </div>
+          <ul class="user-menu">
+            <li>
```
нужно задать класс

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          </div>
+          <ul class="user-menu">
+            <li>
+              <a href="#">
```
нужно задать класс

GasparColor replied:
не сделано

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+                <img src="images/icons/in_home.svg" alt="На главную." class="breadcrumbs-image">
+              </a>
+            </li>
+            <li>
```
это декоративный элемент, не нужно его делать отдельной лишкой, его сделаем стилями

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          <section>
+            <h2 class="visually-hidden">Список гостиниц с фильтрами</h2>
+            <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+              <fieldset>
```
нужно задать класс

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+        </nav>
+      </header>
+      <main class="main-inner">
+        <header class="inner-header">
```
не нужен тут тег, но даже если его оставлять, он должен перед формой закрываться

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          </section>
+        </header>
+        <section class="sorting" >
+          <h2 class="found-hotels">Найдено гостиниц: 38</h2>
```
цифры лучше обернуть в спан, а название класса нужно сделать универсальным, ведь эта разметка может использоваться на других страницах, где будут не гостиницы, а например другие услуги или товары

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+        <section class="sorting" >
+          <h2 class="found-hotels">Найдено гостиниц: 38</h2>
+          <select class="select-control" aria-label="Сортировка">
+          <option value="cheap">Сначала дешёвые</option>
```
у вложенных тегов должен быть отступ слева

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          <option value="expensive">Сначала дорогие</option>
+          </select>
+          <ul class="type-of-catalog">
+            <li>
```
нужно задать класс

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          </select>
+          <ul class="type-of-catalog">
+            <li>
+              <a href="#" class="button-black">
```
нужно задать класс связанный с названием списка, например type-of-catalog-link

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+                <img src="/images/product-card/hotel-1.jpg" width="300" height="212" class="product-card-image" alt="Отель Amara Resort & Spa.">
+              </a>
+              <a class="product-card-link" href="#">
+                <h3 class="name-hotel">Amara Resort & Spa</h3>
```
по аналогии тут должно быть название класса product-card-title

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+                <h3 class="name-hotel">Amara Resort & Spa</h3>
+              </a>
+              <span class="product-card-type">Гостиница</span>
+              <span class="price">От 4000 ₽</span>
```
и тут по аналогии priduct-card-price

GasparColor replied:
название класса должно быть максимально понятным, у элемента в названии класса должно быть в привязка к блоку, в котором этот элемент находится, это ведь не просто самостоятельны элемент с ценой, это цена внутри карточки товара

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+              </a>
+              <span class="product-card-type">Гостиница</span>
+              <span class="price">От 4000 ₽</span>
+              <a href="#" class="product-card button ">Подробнее</a>
```
product-card-details-link

Statsou replied:
Могу ли я оставить второй класс button для стиля кнопки?

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+              <span class="product-card-type">Гостиница</span>
+              <span class="price">От 4000 ₽</span>
+              <a href="#" class="product-card button ">Подробнее</a>
+              <button class="favorites-off">Избранное</button>
```
product-card-button

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+              <span class="price">От 4000 ₽</span>
+              <a href="#" class="product-card button ">Подробнее</a>
+              <button class="favorites-off">Избранное</button>
+              <span class="stars">
```
не нужно список в спан оборачивать

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+              <a href="#" class="product-card button ">Подробнее</a>
+              <button class="favorites-off">Избранное</button>
+              <span class="stars">
+                <ul>
```
product-card-stars

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+              <button class="favorites-off">Избранное</button>
+              <span class="stars">
+                <ul>
+                  <li>
```
задать класс

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+              <span class="stars">
+                <ul>
+                  <li>
+                  <img src="/images/icons/raiting-star.svg" class="star" alt="Оценка качества.">
```
product-card-stars-image

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+            </li>
+          </ul>
+        </section>
+        <section class="subscribe-title-catalog">
```
просто subscribe или scubscription

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          <p class="subscribe-text">Подпишитесь на рассылку</p>
+          <p class="subscribe-description">Только полезная информация и никакого спама,<br> честное бойскаутское!</p>
+          <form action="https://echo.htmlacademy.ru/" method="post">
+              <input type="email" name="email" placeholder="Ваш e-mail" class="email">
```
subscribe-email или лучше subscribe-input

GasparColor replied:
и тут лишний отступ слева

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          <p class="subscribe-description">Только полезная информация и никакого спама,<br> честное бойскаутское!</p>
+          <form action="https://echo.htmlacademy.ru/" method="post">
+              <input type="email" name="email" placeholder="Ваш e-mail" class="email">
+              <button type="submit" value="Подписаться" name="e-mail" class="subscribe-button">Подписаться</button>
```
не нужен тут атрибут value

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          </form>
+        </section>
+      </main>
+      <footer class="social">
```
это футер, а не social

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+        </section>
+      </main>
+      <footer class="social">
+        <ul class="contact-messengers">
```
а это список social

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+      <footer class="social">
+        <ul class="contact-messengers">
+          <li class="social-item">
+            <a href="https://vk.com/htmlacademy" class="social-link-vk"><span class="visually-hidden">Перейти на на сайт В Контакте.</span></a>
```
у ссылок внутри этого списка должен быть один общий класс

GasparColor commented on catalog.html:
```diff
@@ -0,0 +1,302 @@
+          </li>
+        </ul>
+        <a href="tel:+7(812)8121212" class="contact-phone">+7 (812) 812-12-12</a>
+        <a href="https://htmlacademy.ru" class="academy">
```
что значит класс academy ? он не передает сути этого блока, тут можно задать класс copyright

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+      <main class="main-page">
+        <div class="welcome">
+          <h1 class="visually-hidden">Добро пожаловать в Седону</h1>
+          <img src="images/welcome/welcome_sedona.svg" width="458" height="352" alt="Добро пожаловать в Седону." class="img-welcome">
```
welcome-image

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+          <img src="images/welcome/welcome_sedona.svg" width="458" height="352" alt="Добро пожаловать в Седону." class="img-welcome">
+        </div>
+        <section class="main-headline">
+          <h2 class="headline text">Седона — небольшой городок в Аризоне,<br>
```
main-headline-title

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+        <section class="main-headline">
+          <h2 class="headline text">Седона — небольшой городок в Аризоне,<br>
+          заслуживающий большего!</h2>
+          <p class="headline description">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
```
main-headline-description

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+          заслуживающий большего!</h2>
+          <p class="headline description">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+        </section>
+        <ul class="advantages-title">
```
advantages-list

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+          <p class="headline description">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+        </section>
+        <ul class="advantages-title">
+          <li class="advantages-items">
```
advantages-list-item

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+        </section>
+        <ul class="advantages-title">
+          <li class="advantages-items">
+            <div class="advantage blue">
```
advantages-list-inner

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+        <ul class="advantages-title">
+          <li class="advantages-items">
+            <div class="advantage blue">
+              <h3 class="advantages-text-blue">Настоящий Городок</h3>
```
advantages-list-title

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+          <li class="advantages-items">
+            <div class="advantage blue">
+              <h3 class="advantages-text-blue">Настоящий Городок</h3>
+              <p class="advatages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
```
advantages-list-description

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+            <div class="advatage-image">
+              <img src="images/card/card-1.jpg" width="800" height="384" alt="Настоящий городок в Седоне." width="800" height="384">
+            </div>
+            <ul class="inner-list">
```
просто inner-list слишком абстрактное название класса, нужно задать такой класс который будет передавать суть этого списка

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+              <img src="images/card/card-1.jpg" width="800" height="384" alt="Настоящий городок в Седоне." width="800" height="384">
+            </div>
+            <ul class="inner-list">
+              <li class="advantage white-12">
```
в названии класса нужно привязаться к классу списка

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+              <h3 class="advantages-text-blue">Там есть <br> мост дьявола</h3>
+              <p class="advatages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+            </div>
+            <ul class="inner-list">
```
этот список не похож на остальные, ему нужно свой класс задать, и от этого класса задать классы вложенным элементам

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+            </div>
+            <ul class="inner-list">
+              <li class="advantage white-12">
+                <img src="images/card/icons/house.svg" width="76" height="73" alt="Жильё." class="icon-grid">
```
это декоративные картинки, сделаем потом стилями, из разметки их нужно удалить

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+          </li>
+        </ul>
+        <div class="search-hotel">
+          <p class="search-hotel text">Заинтересовались?</p>
```
search-hotel-text

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+        </ul>
+        <div class="search-hotel">
+          <p class="search-hotel text">Заинтересовались?</p>
+          <p class="search-hotel description">Укажите предполагаемые даты поездки,<br>
```
search-hotel-description

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+          <p class="search-hotel text">Заинтересовались?</p>
+          <p class="search-hotel description">Укажите предполагаемые даты поездки,<br>
+            и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          <a href="href="catalog.html" class="button searh-hotel">Поиск гостиницы в Седоне</a>
```
search-hotel-link

GasparColor commented on index.html:
```diff
@@ -3,10 +3,146 @@
+            и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          <a href="href="catalog.html" class="button searh-hotel">Поиск гостиницы в Седоне</a>
+        </div>
+        <section class="subscribe-title">
```
просто subscribe , это ведь класс для секции, это не заголовок

Statsou committed:
Исправляет классы

Statsou committed:
Добавляет отступы

GasparColor commented on catalog.html:
```diff
@@ -44,17 +44,14 @@
-        <header class="inner-header">
-          <h1 class="inner-header-title">Гостиницы Седоны</h1>
+        <div class="wrapper-filter">
+          <h1 class="wrapper-filter-title">Гостиницы Седоны</h1>
```
это не заголовок враппера фильтров

GasparColor commented on catalog.html:
```diff
@@ -44,17 +44,14 @@
       <main class="main-inner">
-        <header class="inner-header">
-          <h1 class="inner-header-title">Гостиницы Седоны</h1>
+        <div class="wrapper-filter">
```
это враппер не только фильтров

GasparColor commented on catalog.html:
```diff
@@ -64,7 +61,7 @@ <h1 class="inner-header-title">Гостиницы Седоны</h1>
             <h2 class="visually-hidden">Список гостиниц с фильтрами</h2>
             <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
-              <fieldset>
+              <fieldset class="catalog-filter-checkbox">
```
задай всем филдетам класс - catalog-filter-fieldset или catalog-filter-group

GasparColor commented on catalog.html:
```diff
@@ -114,27 +111,27 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами<
           <ul class="type-of-catalog">
-            <li>
-              <a href="#" class="button-black">
+            <li class="type-of-item">
```
`type-of-catalog-item`

GasparColor commented on catalog.html:
```diff
@@ -147,106 +144,98 @@ <h2 class="found-hotels">Найдено гостиниц: 38</h2>
-              </span>
+              <span class="product-card-price">От 4000 ₽</span>
+              <a href="#" class="product-card-details-link button">Подробнее</a>
+              <button class="product-card-button">Избранное</button>
```
у кнопки неправильный тип

GasparColor commented on catalog.html:
```diff
@@ -271,29 +260,29 @@ <h3 class="name-hotel">GreenTree Inn</h3>
           </ul>
         </section>
-        <section class="subscribe-title-catalog">
+        <section class="subscription">
```
или тут нужно сделать класс subscribe , или везде ниже задать классы от класса subscription , а не от subscribe как сейчас

GasparColor commented on index.html:
```diff
@@ -46,100 +46,97 @@
         </div>
-        <section class="main-headline">
-          <h2 class="headline text">Седона — небольшой городок в Аризоне,<br>
+        <section class="main-headline-title">
```
это не заголовок

GasparColor commented on index.html:
```diff
@@ -46,100 +46,97 @@
-        <section class="main-headline">
-          <h2 class="headline text">Седона — небольшой городок в Аризоне,<br>
+        <section class="main-headline-title">
+          <h2 class="main-headline-text">Седона — небольшой городок в Аризоне,<br>
```
а это заголовок

GasparColor commented on index.html:
```diff
@@ -46,100 +46,97 @@
+              <h3 class="advantages-list-title">Настоящий Городок</h3>
+              <p class="advantages-list-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
             </div>
             <div class="advatage-image">
```
advantages-list-image-wrapper

GasparColor commented on index.html:
```diff
@@ -46,100 +46,97 @@
-                <h3 class="advantages-text">Небольшая площадь</h3>
-                <p class="advatages-description">Все достопримечательности находятся очень близко</p>
+            <ul class="nested-list-advantages">
+              <li class="nested-list-item advantage-12">
```
не нужны цифры в названии классов. nested-list-advantages-item

GasparColor commented on index.html:
```diff
@@ -46,100 +46,97 @@
-                <p class="advatages-description">Все достопримечательности находятся очень близко</p>
+            <ul class="nested-list-advantages">
+              <li class="nested-list-item advantage-12">
+                <h3 class="nested-item-text">Небольшая площадь</h3>
```
nested-list-advantages-title и ниже по аналогии

GasparColor commented on index.html:
```diff
@@ -46,100 +46,97 @@
-                <img src="images/card/icons/house.svg" width="76" height="73" alt="Жильё." class="icon-grid">
-                <h3 class="advantages-text under-icon">Жильё</h3>
-                <p class="advatages-description">Рекомендуем пожить <br> в настоящем мотеле, <br>
+            <ul class="nested-list-images">
```
в названии класса не нужно привязываться к внешнему виду. это сейчас в макете нарисовали картинки, а завтра придет менеджер и скажет давайте их уберем

Statsou committed:
Исправляет списки классов
